index.bs

```diff
@@ -489,8 +489,8 @@ AV1 Metadata sample group entry {#metadatasamplegroupentry}
 <pre class="def">
 	Group Type: <dfn value noexport for="AV1MetadataSampleGroupEntry">av1M</dfn>
 	Container:  Sample Group Description Box ('sgpd')
-	Mandatory:  No
-	Quantity:   Zero or more.
+	Mandatory:  <assert>No</assert>
+	Quantity:   <assert>Zero or more</assert>.
 </pre>
 
 
```

Comment:
Line 521 contains two occurrences of `SHALL` but is not wrapped with the `<assert>` tags:

`<dfn export>metadata_specific_parameters</dfn> provides an additional part of the <code>grouping_type_parameter</code>, which MAY be used to distinguish different sample groups having the same <code>metadata_type</code> but different sub-parameters. In this version of the specification, <code>metadata_specific_parameters</code> is only defined when <code>metadata_type</code> is set to <code>METADATA_TYPE_ITUT_T35</code> in which case its value SHALL be set to the first 24 bits of the <code>metadata_itut_t35</code> structure. For other types of metadata, its value SHOULD be set to 0. In all cases, when the <code>grouping_type_parameter</code> is used, readers processing sample groups SHALL use the entire value of <code>grouping_type_parameter</code>.`